package.json

```diff
@@ -21,13 +21,14 @@
     "test:watch": "mocha --recursive --watch test",
     "test:karma": "karma start",
     "test:env": "sh ./example-test.sh",
-    "test:all": "npm run react:13 && npm run test:only && npm run react:14 && npm run test:only && npm run react:15.4 && npm run test:only && npm run react:15 && npm run test:only",
+    "test:all": "npm run react:13 && npm run test:only && npm run react:14 && npm run test:only && npm run react:15.4 && npm run test:only && npm run react:15 && npm run test:only && npm run react:16 && npm run test:only",
     "clean-local-npm": "rimraf node_modules/.bin/npm node_modules/.bin/npm.cmd",
     "react:clean": "npm run clean-local-npm && rimraf node_modules/react node_modules/react-dom node_modules/react-addons-test-utils node_modules/react-test-renderer && npm prune",
```

Comment:
doesn't create-react-class not need to be removed because it'd jut be ignored in older versions?